common/smoke-test/run.js

```diff
@@ -6,12 +6,16 @@ async function main() {
   const manifest = require("./run-manifest.json");
 
   let exitCode = 0;
-  let samplesToExecute = [];
-  let failures = [];
+  const samplesToExecute = [];
+  const failures = [];
 
   // Bring all samples and includes into memory
-  for (let entry of manifest) {
-    console.log(`Importing samples for ${entry.Name}...`);
+  for (const entry of manifest) {
```

Comment:
Should we include a TODO here to start running the ARM samples as well at some point?

Reply:
I wasn't sure about arm- samples.  @praveenkuttappan do you know whether we want them to run as part of smoke tests

Reply:
@ckairen should we exclude ARM samples when generating the run manifest json, or is it also desirable to include them?